### DISCUSSION TITLE
Logging for python functions

### DISCUSSION THREAD

**Discussion by aditya2803** [vhive-serverless/vSwarm#200] [General]
Hi, I went through [this](https://github.com/ease-lab/vSwarm/blob/4a66400ce45d8bf8d5fe2c84598e99e8b6630f8a/docs/adding_benchmarks.md#logging) link for logging within the functions, but seems likes this is only applicable for Golang. I wanted to know if similar packages are available for logging within functions in python ? (like pyaes, helloworld etc provided in vHive)

**Comment by ustiugov** [comment]
You are free to use any convenient logging library for other languages. We just wanted to unify these decisions for Golang because Go is widely used in container infrastructure. Please prioritize the default or widely used logging modules. [An example for Python.](https://github.com/ease-lab/vSwarm/blob/4ea1d7b6a9caad67d7596342d031dce888755e7f/benchmarks/video-analytics/object_recognition/recog.py#L47)

**Reply by aditya2803** [reply]
Great, this worked perfectly, thanks !
